platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/fixtures/ToolingApiSpec.groovy

```diff
@@ -41,6 +44,22 @@ trait ToolingApiSpec {
         return (ToolingApiBackedGradleExecuter) getExecuter()
     }
 
+    TestFile getProjectDir() {
+        temporaryFolder.testDirectory
+    }
+
+    BuildTestFile singleProjectBuildInRootFolder(String projectName, @DelegatesTo(BuildTestFile) Closure cl = {}) {
```

Comment:
💅  I'm wondering, should we avoid this ambiguity `dir` <-> `folder` in naming here? Given that underlying API is using `dir` term, probably we should use it in this new methods as well?

Reply:
It's a good point. I initially copied these helpers from another `ToolingApiSpecification` fixture